pkg/build/interfaces.go

```diff
@@ -49,6 +49,12 @@ type Ignorer interface {
 	Ignore(*api.Config) error
 }
 
+// Runner allows the user to actually run the produced imgae as part of the s2i
+// cmd line invocation, to see if it operates as he would expect
+type Runner interface {
+	Run(*api.Config) error
+}
```

Comment:
There's no need to add this interface here. First of all you're not using it anywhere inside the build. Moreover it is not related to build in any way.